samza-core/src/test/java/org/apache/samza/container/grouper/task/TestGroupByContainerCount.java

```diff
@@ -30,19 +30,27 @@
 import org.apache.samza.job.model.TaskModel;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
```

Comment:
Prefer not using PowerMock. Can you make this a protected method that's mocked in test?

Reply:
I agree this is better than Powermock, but this mocking should be deleted in my next patch(as part of SEP-11). Since this is a short-lived, not sure the value of changing it in right way.

Reply:
Sounds good, let's remove this in the follow up PR.